src/test/java/core/basesyntax/validation/DataValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.validation;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DataValidatorTest {
+    private final DataValidator dataValidator = new DataValidator();
+
+    @Test
+    public void checkForEmptyList_validate_Ok() {
+        List<String> inputData = Arrays.asList("type,fruit,quantity");
+        boolean isCorrectHeader = dataValidator.validate(inputData);
+
+        Assertions.assertTrue(isCorrectHeader);
+    }
+
+    @Test
+    public void invalidQuantityThrowException_validate_NotOk() {
+        List<String> inputData = Arrays.asList(
+                "type,fruit,quantity",
+                "b,banana,abc"
+        );
+
+        Assertions.assertThrows(IllegalArgumentException.class, () ->
+                dataValidator.validate(inputData));
+    }
+
+    @Test
```

Comment:
Consider using descriptive test names following given/when/then style for better readability. For example, `givenInvalidHeader_whenValidate_thenThrowsException`.